apps/journeys-admin/src/components/JourneyList/JourneyCard/JourneyCard.tsx

```diff
@@ -1,23 +1,102 @@
 import { ReactElement } from 'react'
-import { Card, Chip, Typography } from '@mui/material'
+import moment from 'moment'
 import { GetJourneys_journeys as Journey } from '../../../../__generated__/GetJourneys'
+import { Card, Chip, Typography, Box } from '@mui/material'
+import CheckCircleIcon from '@mui/icons-material/CheckCircle'
+import EditIcon from '@mui/icons-material/Edit'
+import TranslateIcon from '@mui/icons-material/Translate'
+
 import JourneyCardMenu from './JourneyCardMenu'
+import { AccessAvatars } from './AccessAvatars/AccessAvatars'
+import { user1, user2, user3 } from './AccessAvatars/AccessAvatarsData'
 
 interface JourneyCardProps {
   journey: Journey
 }
 
 const JourneyCard = ({ journey }: JourneyCardProps): ReactElement => {
+  const AccessAvatarsProps = { users: [user1, user2, user3] }
+  const date =
+    moment(journey.createdAt).format('YYYY') === moment().format('YYYY')
+      ? moment(journey.createdAt).format('MMM Do')
+      : moment(journey.createdAt).format('MMM Do, YYYY')
+
   return (
-    <Card>
-      {/* Update according to wireframe */}
-      <Typography>{journey.title}</Typography>
-      {/* <Typography>{journey.createdAt}</Typography> */}
-      <Typography>{journey.description}</Typography>
-      {/* AccessAvatars */}
-      {/* <Chip label={journey.status }/> */}
-      <Chip label={journey.locale.substr(0, 2)} />
-      <JourneyCardMenu />
+    <Card sx={{ padding: '15px 29px', borderRadius: '0px' }} variant="outlined">
```

Comment:
```suggestion
    <Card sx={{ padding: '15px 29px', borderRadius: '0px' }} variant="outlined">
```
Avoid hardcoding pixel values - we have spacing tokens we can use for padding. See https://mui.com/system/the-sx-prop/#spacing